postfix/datadog_checks/postfix/postfix.py

```diff
@@ -176,8 +176,9 @@ def _get_queue_count(self, directory, queues, tags):
                 count = sum(len(files) for root, dirs, files in os.walk(queue_path))
             else:
                 # can dd-agent user run sudo?
-                test_sudo = os.system('setsid sudo -l < /dev/null')
```

Comment:
How was that supposed to redirect **to** `/dev/null` ?

Reply:
🤷‍♂️